crates/pop-contracts/src/call/metadata.rs

```diff
@@ -68,6 +81,39 @@ fn process_args(message_params: &[MessageParamSpec<PortableForm>]) -> Vec<Param>
 	args
 }
 
+/// Generates a list of processed argument values for a specified contract message,
+/// wrapping each value in `Some(...)` or replacing it with `None` if the argument is optional
+///
+/// # Arguments
+/// * `path` -  Location path of the project.
+/// * `message_label` - Label of the contract message to retrieve.
+/// * `args` - Argument values provided by the user.
+pub fn generate_message_args(
+	path: Option<&Path>,
```

Comment:
Why option? Make the caller specify so that this function is less complex.

"In public interfaces, you usually don’t want to use Path or PathBuf directly, but rather a generic P: AsRef<Path> or P: Into<PathBuf>. That way the caller can pass in Path, PathBuf, &str or String."

Reply:
Nice feedback, thanks